docs/docs/cmd/spo/homesite/homesite-list.mdx

```diff
@@ -0,0 +1,104 @@
+import Global from '/docs/cmd/_global.mdx';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# spo homesite list
+
+Lists all home sites
+
+## Usage
+
+```sh
+m365 spo homesite list [options]
+```
+
+## Options
+
+<Global />
+
+## Examples
+
+List all the home sites
+
+```sh
+m365 spo homesite list
+```
+
+## Response
+
+<Tabs>
+  <TabItem value="JSON">
+
+  ```json
+  {
+    "value": [
+      {
+        "Audiences": [
+          {
+            "Email": "ColumnSearchable@contoso.onmicrosoft.com",
+            "Id": "978b5280-4f80-47ea-a1db-b0d1d2fb1ba4",
+            "Title": "ColumnSearchable Members"
+          },
+          {
+            "Email": "contosoteam@contoso.onmicrosoft.com",
+            "Id": "21af775d-17b3-4637-94a4-2ba8625277cb",
+            "Title": "Contoso TeamR Members"
+          }
```

Comment:
For arrays we require to display only 1 item so we don't clutter the page too much. So I suggest we remove one audience object.